include/deal.II/base/parameter_handler.h

```diff
@@ -901,8 +902,13 @@ class ParameterHandler : public Subscriptor
 
   /**
    * Constructor.
+   *
+   * If @p skip_undefined is <code>true</code>, the parameter handler
+   * will skip undefined sections and entries. This is usefull for partially
+   * parsing a parameter file, for example to obtain only the spatial dimension
+   * of the problem.
```

Comment:
I feel that it would be useful to reinforce that the default behaviour is that all subsections are expected to exist.

Reply:
Something like this: By default all entries and subsections are expected to be declared.
?